Preparation/Sand_Clock/Program.cs

```diff
@@ -7,17 +8,48 @@ internal class Program
         static void Main(string[] args)
         {
             Print();
-            string x = Console.ReadLine();
-            string y = Console.ReadLine();
 
-            
+            SandCLock();
         }
 
-        static void Print()
+
+        static void SandCLock()
         {
-            Console.WriteLine("Sand Clock");
-            TimeSpan.FromSeconds(10);
-            Console.WriteLine("Enter value: ");
+            int a = int.Parse(Console.ReadLine());
+            int b = int.Parse(Console.ReadLine());
```

Comment:
Maybe some hint for the user to what to enter? For now - leave as is. For future I also suggest you read parameters out of the "business logic" function and pass them as args.